wicket-util/src/main/java/org/apache/wicket/util/resource/XSLTResourceStream.java

```diff
@@ -102,6 +99,16 @@ public XSLTResourceStream(final IResourceStream xsltResource, final IResourceStr
 		}
 	}
 
+	/**
+	 * Used to provide a configured {@link javax.xml.transform.TransformerFactory}.
+	 *
+	 * @return the transformer factory
+	 */
+	protected javax.xml.transform.TransformerFactory getTransformerFactory()
+	{
+		return javax.xml.transform.TransformerFactory.newInstance();
```

Comment:
IMO, if we are going to make this configurable by the user than we should use a more secure factory by default  and let the apps to go back to unsecure if needed.